dispatcher/templates/candidate_applied.jinja2

```diff
@@ -0,0 +1,82 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+{% import "snippets/macros.jinja2" as macros %}
+{{ macros.head("MyAEGEE: New candidature submitted") }}
```

Comment:
Since these are not being used, I have not reviewed them yet

Reply:
i think they are only used by the sender?

Reply:
I made a related comment, we're using the subjects from constants instead of this title